fem/linearform_ext.cpp

```diff
@@ -104,6 +105,8 @@ void LinearFormExtension::Assemble()
       }
 
       // Assemble the linear form
+      Vector bdr_b(bdr_restrict_lex->Height(),
+                   Device::GetDeviceTemporaryMemoryType());
```

Comment:
Same as above. There are probably other similar situations that need to be handled.